schemas/context/experienceevent.schema.json

```diff
@@ -18,7 +18,13 @@
   "description": "An ExperienceEvent captures observations, including the point in time and identity of the subject involved. Experience Events are fact records of what occurred, representing what happened without aggregation or interpretation. They are critical for time-domain analytics as they allow for observation and analysis of changes that occur in a given window of time and the comparison between multiple windows of time to track trends. Experience Events can be either explicit (directly observable human actions) or implicit (raised without a direct human action).",
   "definitions": {
     "experienceevent": {
-      "properties": { }
+      "properties": {
+        "xdm:correlationId": {
+          "title": "ExperienceEvent Correlation ID",
+          "type": "string",
```

Comment:
I would suggest adding `format: "uuid"` to constrain the format, guide users to actually use a UUID, and reduce the chance of users from accidentally sending in junk like "1" or "test".

Reply:
Brandon Pack - Do you agree with this recommendation ?

Reply:
Yes, I agree with this.  Do we need to be more explicit and say UUID v4?

Reply:
added